src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -1596,6 +1624,714 @@ struct npc_silverpine_forsaken_bat : public VehicleAI
+    {
+        if (Unit* caster = GetCaster())
+        {
+            _playerGUID = caster->GetGUID();
+
+            for (Position const& pos : ForsakenTrooperMPos)
+                caster->SummonCreature(NPC_FORSAKEN_TROOPER_F_LORDAERON, pos, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID);
+
+            for (Position const& pos : ForsakenTrooperFPos)
+                caster->SummonCreature(NPC_FORSAKEN_TROOPER_M_LORDAERON, pos, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID);
+
+            for (Position const& pos : WorgenRenegadePos)
+                caster->SummonCreature(NPC_WORGEN_RENEGADE, pos, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID);
+
+            for (Position const& pos : OrcDemolisherPos)
+                caster->SummonCreature(NPC_ORC_DEMOLISHER_LORDAERON, pos, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID);
+
+            for (Position const& pos : SeaOrcPos)
+                caster->SummonCreature(NPC_ORC_MOVER_LORDAERON, pos, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID);
+
+            if (Creature* leaderOrc = caster->SummonCreature(NPC_ORC_MOVER_LORDAERON, SeaOrcLeaderPos1, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID))
+            {
+                leaderOrc->GetMotionMaster()->MovePath(PATH_ORC_LEADER1, true);
+
+                for (uint8 i = 0; 0 < 4; i++)
+                    if (Creature* orc = caster->SummonCreature(NPC_ORC_MOVER_LORDAERON, SeaOrcLeaderPos1, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID))
+                        orc->GetMotionMaster()->MoveFollow(leaderOrc, float(3 * i), 0.0f);
+            }
+
+            if (Creature* leaderOrc = caster->SummonCreature(NPC_ORC_MOVER_LORDAERON, SeaOrcLeaderPos2, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID))
+            {
+                leaderOrc->GetMotionMaster()->MovePath(PATH_ORC_LEADER2, true);
+
+                for (uint8 i = 0; 0 < 4; i++)
+                    if (Creature* orc = caster->SummonCreature(NPC_ORC_MOVER_LORDAERON, SeaOrcLeaderPos2, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID))
+                        orc->GetMotionMaster()->MoveFollow(leaderOrc, float(3 * i), 0.0f);
+            }
+
+            for (Position const& pos : DreadguardPos)
+                caster->SummonCreature(NPC_DREADGUARD_LORDAERON, pos, TEMPSUMMON_TIMED_DESPAWN, 300s, 0, SPELL_SUMMON_LORDAERON_ACTORS, _playerGUID);
+        }
+    }
+
+    void Register() override
+    {
+        OnEffectHitTarget += SpellEffectFn(spell_silverpine_summon_lordaeron_actors::HandleScriptEffect, EFFECT_0, SPELL_EFFECT_SCRIPT_EFFECT);
+    }
+
+private:
+    ObjectGuid _playerGUID;
+};
+
+// 84173 - Despawn All Summons
+class spell_silverpine_despawn_all_summons_lordaeron : public SpellScript
```

Comment:
Maybe create a generic spell script, there's like 100 similar spells, spell_gen_despawn_owned or something like that

Reply:
I'll consider it, but I'm most likely taking that approach since we can control target conditions through DB.

Reply:
I have added this spell as generic in an older PR which is soon getting merged.